elliptic-curve/src/hash2field/expand_msg_xmd.rs

```diff
@@ -0,0 +1,72 @@
+use super::ExpandMsg;
+use core::marker::PhantomData;
+use digest::{
+    generic_array::{typenum::Unsigned, GenericArray},
+    BlockInput, Digest,
+};
+use subtle::{Choice, ConditionallySelectable};
+
+/// Placeholder type for implementing expand_message_xmd based on a hash function
+#[derive(Debug)]
+pub struct ExpandMsgXmd<HashT> {
+    phantom: PhantomData<HashT>,
+}
```

Comment:
Not sure I quite understand the purpose of this

Reply:
Oh wait, I see, it's to make the blanket impl work.

Curious if there's a better way to get this done, but in the interests of upstreaming I guess this is probably fine for now.